go.mod

```diff
@@ -56,7 +56,7 @@ require (
 	github.com/gorilla/mux v1.8.1
 	github.com/grafana/alloy-remote-config v0.0.9
 	github.com/grafana/alloy/syntax v0.1.0
-	github.com/grafana/beyla v0.0.0-20250108110233-3f1b9b55c6dc // custom beyla 1.9.6 branch without breaking replace directives
+	github.com/grafana/beyla v1.9.1-0.20250122195759-1117708def46 // custom beyla 1.10 branch without git lfs
```

Comment:
Any reason why this tag 1.9.1 still?

Reply:
Custom commits generally pull the last released tag in packages

Reply:
Thought it really should be 1.9.8